src/ui/public/agg_response/tabify/_buckets.js

```diff
@@ -68,7 +68,9 @@ TabifyBuckets.prototype._isRangeEqual = function (range1, range2) {
 TabifyBuckets.prototype._orderBucketsAccordingToParams = function (params) {
   if (params.filters && this.objectMode) {
     this._keys = params.filters.map(filter => {
-      return filter.label || filter.input.query || '*';
+      const query = _.get(filter, 'input.query.query_string.query', filter.input.query);
+      const queryString = typeof query === 'string' ? query : JSON.stringify(query);
```

Comment:
are we sure query will never be null ?

Reply:
It could only ever be null if the user manually edits their saved object to make it so and even then, I can't think of a better action than stringifying it. We'll report it as an invalid query elsewhere so there's no need to throw an error here I think.